packages/vuetify/src/components/VTimeline/_variables.scss

```diff
@@ -1,18 +1,39 @@
-@import '../../styles/styles.sass';
-
-$timeline-divider-center: 50% !default;
-$timeline-divider-width: 96px !default;
-
+// Defaults
+$timeline-density-comfortable-grid-template-end: min-content min-content auto !default;
+$timeline-density-comfortable-grid-template-start: auto min-content min-content !default;
+$timeline-density-compact-grid-template-end: min-content auto !default;
+$timeline-density-compact-grid-template-start: auto min-content !default;
+$timeline-divider-dot-elevation: 0 !default;
+$timeline-divider-line-background: rgba(var(--v-border-color), var(--v-border-opacity)) !default;
+$timeline-divider-line-horizontal-width: var(--v-timeline-line-inset, 0px) !default;
+$timeline-divider-line-thickness: var(--v-timeline-line-thickness) !default;
+$timeline-dot-border-radius: 50% !default;
+$timeline-dot-divider-background: rgb(var(--v-theme-on-surface)) !default;
```

Comment:
You can't use on color here, it won't work together with icons in the dot.

Reply:
Up for any suggestions. The dot should use surface color from theme.

Reply:
It can use the surface color. Not the surface on color.

Reply:
(also if it's supposed to be the surface color why can't default value for `dot-color` just be `surface`?)

Reply:
As a matter of process with assigning default colors via prop. That in itself is a larger discussion.